languoids/tree/atla1278/volt1241/benu1247/benu1248/taro1265/kwan1287/vagh1247/biji1246/md.ini

```diff
@@ -3,8 +3,11 @@
 name = Bijim
 level = dialect
 macroareas = 
-	Africa
+	UNKNOWN
```

Comment:
What is this supposed to mean? Since we don't have `UNKNOWN` in `config/macroareas.ini`, this fails the checks.